criterion-measurement/stack.yaml

```diff
@@ -0,0 +1,8 @@
+resolver: lts-10.5
```

Comment:
Why do you need a separate `stack.yaml` file just for `criterion-measurement`? Doesn't the top-level one include this information already?

Reply:
I understand it's replicated information, but I'd rather keep this stack.yaml in case other downstream libraries need to build on it.

Reply:
I don't understand. If anyone wants to clone `criterion` locally and use `stack` to build it, then they'll use the `stack.yaml` from the top-level directory, yes? Why is this ever needed?

Reply:
I thought this might be needed in order to build `criterion-measurement` as a standalone library? Or, I guess I didn't consider the possibility you're mentioning.

Reply:
I'm not sure we're on the same page here, so let me try and contextualize this.

First, we never ship `stack.yaml`/`cabal.project` files in sdists that are uploaded to Hackage. The only reason one would ever use them is if one were to check out `criterion` locally on build it from source.

With that in mind, one should only ever need the `stack.yaml` file in the top-level directory, as it has enough information to build `criterion`, `criterion-measurement`, or both (depending on what targets you specify).

In light of this, the `stack.yaml` file in `criterion-measurement/` is entirely redundant. All of the information that `stack` needs to build it is already contained in the top-level `stack.yaml` file. Does this make sense?

Reply:
Absolutely, thank you; I'll remove the additional stack.yaml.